build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CreateAndroidEmulator.cs

```diff
@@ -47,7 +47,7 @@ public override bool Execute ()
 			}
 
 			var env = new List<string> ();
-			env.Add ($"ANDROID_SDK_HOME={AvdManagerHome}");
+			env.Add ($"ANDROID_PREFS_ROOT={AvdManagerHome}");
```

Comment:
do we have a link to docs on `$ANDROID_PREFS_ROOT`?

Reply:
It's indirectly mentioned in the link in the description above: https://developer.android.com/studio/command-line/variables